lib/exchange/provider/trocador_exchange_provider.dart

```diff
@@ -333,3 +325,29 @@ class TrocadorExchangeProvider extends ExchangeProvider {
     }
   }
 }
+
+class TrocadorPartners {
+  final String name;
+  final String rating;
+  final double insurance;
+  final bool enabledMarkup;
+  final double eta;
+
+  TrocadorPartners({
+    required this.name,
+    required this.rating,
+    required this.insurance,
+    required this.enabledMarkup,
+    required this.eta,
+  });
+
+  factory TrocadorPartners.fromJson(Map<String, dynamic> json) {
+    return TrocadorPartners(
+      name: json['name'] as String,
+      rating: json['rating'] as String,
+      insurance: json['insurance'] as double,
+      enabledMarkup: json['enabledmarkup'] as bool,
+      eta: json['eta'] as double,
```

Comment:
make all of them nullable, since this is not our API and we are not 100% sure they will keep all those mandatory and returned every time

Reply:
done